### ISSUE TITLE
Hitting Enter should publish Tweet

### ISSUE THREAD

**Issue by GoogleCodeExporter** [emuikernel/wittytwitter#13]
```
What steps will reproduce the problem?
1. Select Update
2. Type a Tweet
3. Hit Return

What is the expected output? What do you see instead?
Would like to see the Tweet published.

What version of the product are you using? On what operating system?
0.1.3

Please provide any additional information below.


```

Original issue reported on code.google.com by `kurtbroc...@gmail.com` on 19 Oct 2007 at 3:49

**Comment by GoogleCodeExporter** [comment]
Original comment by `alan...@gmail.com` on 19 Oct 2007 at 5:13
- Added labels: **Milestone-v0.1.4**, **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
Original comment by `alan...@gmail.com` on 19 Oct 2007 at 5:14
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
fixed in v0.1.4
```

Original comment by `alan...@gmail.com` on 20 Oct 2007 at 11:59
- Changed state: **Fixed**
